solana/program_v2/tests/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@identity.com/gateway_v2-tests",
+  "version": "1.0.0",
+  "description": "gateway_v2 tests",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Hm this package.json is a little stub (because it does not specify the actual test runners and dependencies).

I think the better folder structure is the one from Cryptid:

```
        "packages/client/idl",
        "packages/client/cli",
        "packages/client/core",
        "packages/client/cli",
        "packages/tests"
```

Reply:
THe you need to make sure that the test package is as independent as in shown in Cryptid.